Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Our test scenario is the following:
Go to a project with an Umlaut in a field (for example the title), or change the title to one with an umlaut. then go to another tab, than go back to the tab with the umlaut -> the umlaut is not correctly formatted. it even changes again, if you switch again to another tab and back.